docs/source/index.rst

```diff
@@ -14,7 +14,7 @@ Mypy is a static type checker for Python 3 and Python 2.7.
 
    introduction
    basics
```

Comment:
Maybe rename the "basics" page to "getting started"? It feels slightly more approachable to me.